src/pantry.js

```diff
@@ -9,7 +9,11 @@ class Pantry {
       if (acc[pantryItem.ingredient]) {
         acc[pantryItem.ingredient].amount += parseInt(pantryItem.amount);
       } else {
-        acc[pantryItem.ingredient] = pantryItem;
+        acc[pantryItem.ingredient] =
+        {
+          id: pantryItem.ingredient,
+          amount: parseInt(pantryItem.amount)
```

Comment:
Thanks for helping fix the server data type errors in making it a number (parseInt()) of the pantryItem.amount.